sunpy/data/data_manager/manager.py

```diff
@@ -172,10 +181,30 @@ def get(self, name):
         `KeyError`
             If ``name`` is not in the cache.
         """
-        return pathlib.Path(self._file_cache[name])
+        return pathlib.Path(self._file_cache[name][self._namespace])
 
     def _cache_has_file(self, urls):
         for url in urls:
             if self._cache._get_by_url(url):
                 return True
         return False
+
+    def _get_module(self, func):
+        """
+        Returns the name of the module (appended with a dot) that a function belongs to.
+        This is made into a separate function for the ability to patch in tests.
```

Comment:
```suggestion
        Returns the name of the module (appended with a dot) that a function belongs to.
```

Reply:
For some reason the "Commit suggestion" button is grayed out. Should I add make the changes and commit manually?

Reply:
Sure.